tests/ansible-role-requirements.yml

```diff
@@ -1,5 +1,76 @@
----
+- name: apt_package_pinning
+  src: https://git.openstack.org/openstack/openstack-ansible-apt_package_pinning
+  scm: git
+  version: stable/newton
+- name: etcd # dependency of os_neutron role
+  scm: git
+  src: https://github.com/logan2211/ansible-etcd
+  version: master
+- name: galera_client
+  src: https://git.openstack.org/openstack/openstack-ansible-galera_client
+  scm: git
+  version: stable/newton
+- name: galera_server
+  src: https://git.openstack.org/openstack/openstack-ansible-galera_server
+  scm: git
+  version: stable/newton
+- name: memcached_server
+  src: https://git.openstack.org/openstack/openstack-ansible-memcached_server
+  scm: git
+  version: stable/newton
+- name: openstack_hosts
+  src: https://git.openstack.org/openstack/openstack-ansible-openstack_hosts
+  scm: git
+  version: stable/newton
+- name: openstack_openrc
+  src: https://git.openstack.org/openstack/openstack-ansible-openstack_openrc
+  scm: git
+  version: stable/newton
+- name: lxc_container_create
+  src: https://git.openstack.org/openstack/openstack-ansible-lxc_container_create
+  scm: git
+  version: stable/newton
+- name: lxc_hosts
+  src: https://git.openstack.org/openstack/openstack-ansible-lxc_hosts
+  scm: git
+  version: stable/newton
 - name: pip_install
   src: https://git.openstack.org/openstack/openstack-ansible-pip_install
   scm: git
-  version: master
+  version: stable/newton
+- name: rabbitmq_server
```

Comment:
All the rest, should, IMO go away. 
What are the functional elements we should do? Here you're trying to integrate rpc-maas into OSA, which is fine for me, but it seems carrying a lot of burden: We already have an RPC-O job that does exactly this. We should just change the a-r-r there to point at master of rpc-maas.